tutorials/docker-instantapp/index.mdx

```diff
@@ -6,86 +6,51 @@ content:
-  - step: Connecting to Your Instance
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/docker-instant-app/#-Connecting-to-Your-Instance
-  - step: Building Your First Container
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/docker-instant-app/#-Building-your-First-Container
-  - step: Running the Application
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/docker-instant-app/#-Running-the-Application
 categories:
   - compute
 dates:
-  validation: 2021-05-10
+  validation: 2022-02-14
   posted: 2019-05-13
 ---
 
-## Getting Started with the Docker InstantApp
-
-This page shows you how to use the Docker InstantApp on your cloud instance.
+Docker is an open source containerization platform. It lets developers package their applications into containers, using OS-level virtualization. This page shows you how to create an Instance with Docker pre-installed using the Docker [InstantApp](/compute/instances/concepts/#instantapp), and then how to start using Docker for your containerization projects.
 
 <Message type="requirement">
 
-- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
 - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
 
 </Message>
 
-## Creating and Starting a New Cloud Instance Using the Docker Instantapp
-
-Before starting, click the **Create an Instance** button in the management console.
-
-<Lightbox src="scaleway-docker_create_instance.png" alt="" />
-
-You will land on the server-creation page where you can choose the **Docker** image in the **InstantApps** tab for your Cloud Instance:
-
-<Lightbox src="scaleway-docker_choose_image.png" alt="" />
+## Creating a new Instance with the Docker InstantApp
 
-Choose the instance type, enter additional parameters like tags and click the **Create a new instance** button. This action starts your instance with Docker pre-installed.
+Follow the instructions for [creating an Instance](/compute/instances/how-to/create-an-instance/). When choosing your **Image**, click the **InstantApps** tab and select the **Docker** InstantApp.
 
-## Connecting to Your Instance
+## Connecting to your Instance
 
-Connect to your instance via SSH. To do so, you must have configured an [SSH key pair](/console/my-project/how-to/create-ssh-key) on your local computer and uloaded the public key to your management console.
-
-On Windows, you can use [Putty](/compute/instances/how-to/connect-to-instance/#how-to-connect-from-windows) to connect to your instance.
-
-On Linux or MacOs, you can open a terminal and type the following command to connect to your instance:
-
-```
-john@localhost:~$ ssh -i ~/.ssh/your_private_key root@your_instance_ip
-```
+Follow the instructions for [connecting to your Instance](/compute/instances/how-to/connect-to-instance/).
```

Comment:
maybe this can be included as a step in another section?

Reply:
I've made 'create and connect' one subheading with two steps :-)